.github/workflows/ci.yml

```diff
@@ -400,33 +400,33 @@ jobs:
   create-tarball:
     name: Create source tarball
     if: contains(github.head_ref, 'dist') || contains(github.head_ref, 'extras') || contains(github.ref, 'release-')
-    needs: [build-and-test]
     runs-on: ubuntu-latest
+    outputs:
+        version: ${{ steps.extract_version.outputs.version }}
     steps:
+    - name: Install python and other dependencies
+      run: sudo apt-get --assume-yes install python3-wheel python3-setuptools python3-pip
+    - name: Install meson and ninja
+      run: pip3 install --user meson ninja PyYAML
     - uses: actions/checkout@v2
       with:
           submodules: recursive
     - name: Extract rizin version
       shell: bash
-      run: echo "##[set-output name=branch;]$(python sys/version.py)"
+      run: echo "##[set-output name=version;]$(python sys/version.py)"
       id: extract_version
-    - name: Download capstone
-      run: |
-        ./configure
-        make -C shlr capstone
-        rm -rf shlr/capstone/.git
-        git clean -dxf .
-        rm -rf .git
     - name: Create archive
       run: |
-        cd ..
-        cp -r rizin rizin-${{ steps.extract_version.outputs.branch}}
-        tar cvzf rizin-src.tar.gz rizin-${{ steps.extract_version.outputs.branch}}
-        mv rizin-src.tar.gz rizin/
+        export PATH=${HOME}/.local/bin:${PATH}
+        meson rewrite kwargs set project / version ${{ steps.extract_version.outputs.version }}
```

Comment:
what is this? wow :D

Reply:
BTW, once we kill ACR there is no need for this anymore, we will just keep the project version in `meson.build`.